### PR TITLE
MALZ  to 7

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Mss of Adi Laqma church.

### CODE REVIEW

File: AlaqmaMaryamZadibaKokweh/MALZ002.xml [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+                           <list>
+                               <item xml:id="a1">                                
+                                   <desc type="DonationNote">The note mentions the donor and commissioner Tasfā Māryām and his relatives</desc>
+                                   <q xml:lang="gez">ዝመጽኀፍ፡ ዘወሀባ፡ ተስፋ፡ ማርያም፡ አቡሁ፡ ገብረ፡ ዋህድ፡ እሙ፡ ወለተ። 
+                                       ኪዳን፡ ዘአጽሐፋ፡ በንዋዩ፡ </q>
+                               </item>
+                               
+                               <item xml:id="a2">                                
+                                   <desc type="DonationNote">A lengthy donation note on the penultimate folium, 
+                                       mentioning in part the same persons as Additional note 1, written in red,
```
Comment (eu-genia) [inline AlaqmaMaryamZadibaKokweh/MALZ002.xml:92]: the same persons as `<ref type="item" target="#a1">Additional note 1</a1>`,

File: AlaqmaMaryamZadibaKokweh/MALZ008.xml [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+                       
+                       
+                       <additions>
+                           <list>
+                               <item xml:id="a1">                                
+                                   <desc type="OwnershipNote">The note on the penultimate folium mentions the church ʾAlāqʷǝmā Wazadiba Kʷakʷǝḥ as the owner</desc>
+                               </item>
+
+                               <item xml:id="a2">                                
+                                   <desc type="Unclear">A pen trial of Kidāna Māryām, on the recto of the ultimat folium</desc>
```
Comment (eu-genia) [inline AlaqmaMaryamZadibaKokweh/MALZ008.xml:73]: ultimat**e**
better `last`

File: AlaqmaMaryamZadibaKokweh/MALZ010.xml [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS1012AddiLaqmaMaryam"/>
+                        <idno>MALZ-010</idno>
+                        </msIdentifier>
+                    <msContents>
+                       <summary/>
+                     
+                        <msItem xml:id="ms_i1">                        
+                            <title type="incomplete" ref="LIT2375Synaxa">Synaxarion (the first half of the year)</title>
```
Comment (eu-genia) [inline AlaqmaMaryamZadibaKokweh/MALZ010.xml:34]: LIT2375Synaxa#FirstHalf?

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: AlaqmaMaryamZadibaKokweh/MALZ002.xml [hunk 4]
```diff
@@ -0,0 +1,162 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="MALZ002" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Lectionary of the Holy Week</title>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply AlaqmaMaryamZadibaKokweh/MALZ002.xml:8]: ok

File: AlaqmaMaryamZadibaKokweh/MALZ002.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply AlaqmaMaryamZadibaKokweh/MALZ002.xml:92]: ok

File: AlaqmaMaryamZadibaKokweh/MALZ008.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply AlaqmaMaryamZadibaKokweh/MALZ008.xml:73]: ok

**Pull request merged**